assets/javascripts/kitten/components/cards/contribution-card/components/action.js

```diff
@@ -8,7 +8,6 @@ export const Action = ({ className, ...props}) => {
       borderRadius={4}
       type="submit"
       {...props}
-      className={className}
       className={classNames(
         'k-ContributionCard__action',
         props.className,
```

Comment:
étant donné que tu as `export const Action = ({ className, ...props}) => {`, il n'y a plus de `props.className`

```suggestion
        className,
```

Reply:
Ah oui ok mais c'est pas ce que je voulais faire ! C'est un oublie